x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/SecurityImplicitBehaviorBootstrapCheck.java

```diff
@@ -59,4 +60,9 @@ public BootstrapCheckResult check(BootstrapContext context) {
     public boolean alwaysEnforce() {
         return true;
     }
+
+    @Override
+    public ReferenceDocs referenceDocs() {
+        return ReferenceDocs.SECURITY_MINIMAL_SETUP;
```

Comment:
The failure message also constructs a link to this page, but does so without using `ReferenceDocs` so we are not validating it. Could you fix that place to use `ReferenceDocs.SECURITY_MINIMAL_SETUP` too?